noetic/distribution.yaml

```diff
@@ -7184,8 +7184,8 @@ repositories:
       - rm_common
       - rm_control
       - rm_dbus
-      - rm_description
       - rm_gazebo
+      - rm_description
```

Comment:
Could you explain this?

Reply:
I dont know whether i should delete it, i want to separate rm_description from rm_control. At first i thought it should be deleted and i deleted it in distribution.yaml, but i haven't deleted it in rm_control. So i add it again in distribution.yaml.

Reply:
@ivanpauno

Reply:
Yes, you need to delete it from `rm_control`.
The new entry for `rm_description` can be added in its first `bloom-release`.

The bad thing is that I think all old packages of `rm_description` will be removed, but IIUC there's no way around that.